department-of-veterans-affairs/caseflow
# A11y Adjustments to Tooltip Component

Connects [CASEFLOW-998](https://vajira.max.gov/browse/CASEFLOW-998) and [CASEFLOW-1002](https://vajira.max.gov/browse/CASEFLOW-1002)

### Description
This changes the `Tooltip` component so that it passes the tooltip-related props through to its children rather than providing a wrapping `span` element.

This allows the tooltip to be shown based on events on the actual target element (such as a `button`) rather than on the wrapper; this results in a single tab stop as well as showing the tooltip upon focusing the target element.

### Acceptance Criteria
- [x] Tooltip shows when target element is focused
- [x] Tooltip still shows when target element is hovered w/ mouse
- [ ] Tooltip content is read with screenreader when target element is focused
- [x] Nothing else broke

### Testing Plan
First: run `yarn install` from `client` folder to update to latest `react-tooltip`
1. Go to Case Details page for any AMA appeal for vet ID `54545454`
2. Ensure that tooltip shows (and is read by screenreader) correctly for the `CopyTextButton` for veteran ID
3. Ensure that tooltip shows (and is read by screenreader) correctly for the FNOD badge
4. Ensure that tooltip shows (and is read by screenreader) correctly for the docket type badge
